polliwog/box/test_box.py

```diff
@@ -63,3 +63,21 @@ def test_box_from_points():
 def test_invalid_shape():
     with pytest.raises(ValueError):
         Box(np.array([1.0, 2.0, 3.0]), np.array([-1.0, 1.0, 1.0]))
+
+
+def test_v():
+    box = create_box()
+    assert box.v.shape == (3, 8)
+    expected = np.array(
+        [
+            [1.0, 2.0, 3.0],
+            [4.0, 2.0, 3.0],
+            [1.0, 3.0, 3.0],
+            [1.0, 2.0, 8.0],
+            [4.0, 3.0, 3.0],
+            [1.0, 3.0, 8.0],
+            [4.0, 2.0, 8.0],
+            [4.0, 3.0, 8.0],
+        ]
+    ).T
```

Comment:
Why is the `.T` needed?

Reply:
@algrs ^^

Reply:
Just for clarity of writing the points. Elsewhere we have a convention that points are 3xN, but it's clearer to write the constants as Nx3, so you get one point to a line of code.

Reply:
Hmm, I think that's backwards. Pretty much everywhere else they are Nx3 (lace, polyline), so e.g. `.v[0]` is a vertex.

Reply:
Ah, drat. You're right, of course. Let me swap it around.

Reply:
Fixed. Of course, that begs the question of if we should change these functions to take Nx3 as well. Should we change the api for these two functions to make them take points in the same form as we use elsewhere?

Reply:
Which functions are taking 3xn?

Reply:
Oh haha `find_rigid_transform` and `find_rigid_rotation` 😝 

Yea, probably we should. Want to do it now or in a follow-on?

Reply:
I'll take care of it now